package.json

```diff
@@ -12,36 +12,50 @@
     "react-helmet": "^5.2.0",
     "react-router": "^4.1.2",
     "react-router-dom": "^4.1.2",
-    "react-scripts": "1.0.11",
     "semantic-ui-css": "2.2.12",
     "semantic-ui-react": "^0.75.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "coverage": "react-scripts test --coverage",
+    "test": "cross-env CI=true react-scripts test",
+    "test:coverage": "CI=true react-scripts test --coverage",
```

Comment:
This should be using `cross-env` as well, I imagine.

Reply:
@wacii Fixed~